test/viz/index.spec.ts

```diff
@@ -29,6 +29,7 @@ describe('End-to-end-tests for viz', () => {
       devBucket: 'test/dscc-gen-test-dev',
       prodBucket: 'test/dscc-gen-test-prod',
       yarn: false,
+      codelab: false,
```

Comment:
You should also add a test for codelab: true and make sure it uses the right template. Probably by searching for a string/filename that only exists in one.